### PR TITLE
Fixes failing tests and disabled other locales

### PR DESCRIPTION
**Pull request by flashburst**
- Disabled all locales except english
- Fixed failing tests

### CODE REVIEW

File: i18n.config.js [hunk 1]
```diff
@@ -2,18 +2,18 @@ module.exports = {
...
+      // "ja",
+      // "zh",
+      // "de",
+      // "id",
+      // "it",
+      // "ko",
+      // "ru",
+      // "el",
+      // "tr",
+      // "vi",
```
Comment (mark-15) [inline i18n.config.js:16]: are we really going to remove this? is this intended?

File: src/config/constants.js [hunk 2]
```diff
@@ -29,35 +29,35 @@ export const PoolTypes = {
...
+  // de: "German - Deutsch",
+  // id: "Indonesian - Bahasa Indonesia",
+  // it: "Italian - italiano",
+  // ja: "Japanese - 日本語",
+  // ko: "Korean - 한국어",
+  // ru: "Russian - русский",
+  // es: "Spanish - Español",
+  // el: "Greek - Ελληνικά",
+  // tr: "Turkish - Türkçe",
+  // vi: "Vietnamese - Tiếng Việt",
```
Comment (mark-15) [inline src/config/constants.js:44]: same here are we going to remove this?

File: i18n.config.js (hunk 1 above)
Comment (flashburst) [reply i18n.config.js:16]: Yes, this is intentional. Currently we had the translations from Google Translate which are not good.

File: src/config/constants.js (hunk 2 above)
Comment (flashburst) [reply src/config/constants.js:44]: yes

**Review by mark-15 — APPROVED** [review]

**Pull request merged**